pyfar/dsp/dsp.py

```diff
@@ -1237,8 +1237,14 @@ def find_impulse_response_delay(impulse_response, N=1):
         # in the strict sense, instead of the appriximation implemented in
         # pyfar.
         n_samples = impulse_response.n_samples
-        ir_minphase = sgn.minimum_phase(
-            impulse_response.time[ch], n_fft=4*n_samples)
+
+        # minimum phase warns if the input signal is not symmetric, which is
+        # not critical for this application
+        with warnings.catch_warnings():
```

Comment:
not sure, but I'm just wondering if it would be more elegant if we just have a input flat to ignore the symmetry check like `minimum_phase(h, method='homomorphic', n_fft=None, allow_asymmetry=False)`. Then it can be used whenever we are sure that it does not matter for the case.

Reply:
never mind, I did not notice, that its a function from scipy.

Reply:
We can add a message so that we are sure that we are ignoring the correct warning.
`warnings.filterwarnings("ignore", message="does not appear to by symmetric", type=RuntimeWarning)`

Reply:
Good point. Done as suggested.